From abab75c4567385f2911c578f1fc47be636919802 Mon Sep 17 00:00:00 2001
From: Robert Wiegand <qa2113@motorola.com>
Date: Fri, 22 Jan 2016 11:23:55 -0600
Subject: [PATCH 647/959] IKLOCSEN-2517 dts: Adjust lux coefficients for 8952

Modify the coefficients used to calculate lux values
in the light sensor hub's light sensor.

Change-Id: I1deccf8879a37a369e5ed5e576f2057b1ad8fd9b
Signed-off-by: Robert Wiegand <qa2113@motorola.com>
Reviewed-on: http://gerrit.mot.com/808660
SLTApproved: Slta Waiver <sltawvr@motorola.com>
SME-Granted: SME Approvals Granted
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: George Standish <george.standish@motorola.com>
Reviewed-by: Zhouhao Cui <dbqx86@motorola.com>
Reviewed-by: Jason Jae Hyun Koh <jasonkoh@motorola.com>
Reviewed-by: Daniel Moore <dtmoore@motorola.com>
Reviewed-by: Christopher Fries <cfries@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
---
 arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
index a790d9afe2c..fbf04509b09 100644
--- a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
@@ -543,12 +543,12 @@
 			ct406_recalibrate_threshold = <0x0064>;
 			ct406_pulse_count = <0x03>;
 			ct406_prox_gain = <0x02>;
-			ct406_als_lux1_c0_mult = <0x294>;
-			ct406_als_lux1_c1_mult = <0x55A>;
-			ct406_als_lux1_div = <0x64>;
-			ct406_als_lux2_c0_mult = <0xDA>;
-			ct406_als_lux2_c1_mult = <0x186>;
-			ct406_als_lux2_div = <0x64>;
+			ct406_als_lux1_c0_mult = <0x488>;
+			ct406_als_lux1_c1_mult = <0x888>;
+			ct406_als_lux1_div = <0x80>;
+			ct406_als_lux2_c0_mult = <0x177>;
+			ct406_als_lux2_c1_mult = <0x266>;
+			ct406_als_lux2_div = <0x80>;
 			dsp_iface_enable = <0>;
 			headset_detect_enable = <1>;
 			headset_hw_version = <2>;
-- 
2.11.0

